src/OlStyleParser.spec.ts

```diff
@@ -500,6 +504,9 @@ describe('OlStyleParser implements StyleParser', () => {
           const olTextContent = olText.getText();
           expect(olTextContent).toEqual(dummyFeat.get('name'));
 
+          const olTextRotation = olText.getRotation();
+          expect(olTextRotation).toEqual(expecSymb.rotate * Math.PI / 180);
```

Comment:
Comparing two floating point numbers in JS might lead to unexpected results. I would suggest to make use of Jest's [toBeCloseTo()](https://jestjs.io/docs/en/expect#tobeclosetonumber-numdigits) method here (and overall…).